packages/blocks/github-pr-overview/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "github-pr-overview",
+  "version": "0.0.0",
+  "description": "github-pr-overview block",
```

Comment:
Should this be change

Reply:
Yes please – have a look at the descriptions on https://blockprotocol.org/hub, just a sentence is good, something like "Display the timeline of key events for a GitHub Pull Request"

Reply:
Have updated, I could have sworn when I moved the block in I checked other blocks in `packages/blocks/` and saw they were all "X block" but upon looking again I must have imagined it 😄